### PR TITLE
Add Arvid's NixCon summary

### PR DESCRIPTION
**Pull request by siberov**
🇫🇷

### CODE REVIEW

File: nixcon-2022.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# NixCon 2022: A Brief Summary
+
+NixCon, the yearly convention for all things Nix was held in Paris this year. I went there, I discussed Nix and supply chain security, and I ate baguettes for breakfast, and even for dinner once. This is a brief summary of the event.
+
+Nix is a package manager, first imagined in a soon-to-be twenty years old doctoral dissertation (“The Purely Functional Software Deployment Model”) by Eelco Dolstra in his. It differs from other package managers, such as Debian’s apt or Fedora’s yum, by its strong guarantees of reproducibility; Nix lists all components and dependencies of a software package, not just by name and version, but also with a cryptographic hash of their source code and build environments. This allows consumers of a package to reproduce not only the package’s binary, but also its entire runtime environment. As a consequence, what works on my machine is as good as guaranteed to work on yours.
```
Comment (gabrielskoglund) [inline nixcon-2022.md:5]:
```suggestion
Nix is a package manager, first imagined in a soon-to-be twenty years old doctoral dissertation (“The Purely Functional Software Deployment Model”) by Eelco Dolstra in his. It differs from other package managers, such as Debian’s apt or Fedora’s yum, by its strong guarantees of reproducibility; Nix lists all components and dependencies of a software package, not just by name and version, but also with a cryptographic hash of their source code and build environments. This allows consumers of a package to reproduce not only the package’s binary, but also its entire runtime environment. As a consequence, what works on my machine is as good as guaranteed to work on yours.
```
```suggestion
Nix is a package manager, first imagined in a soon-to-be twenty years old doctoral dissertation (“The Purely Functional Software Deployment Model”) by Eelco Dolstra. It differs from other package managers, such as Debian’s apt or Fedora’s yum, by its strong guarantees of reproducibility. Nix lists all components and dependencies of a software package, not just by name and version, but also with a cryptographic hash of their source code and build environments. This allows consumers of a package to reproduce not only the package’s binary, but also its entire runtime environment. As a consequence, what works on my machine is as good as guaranteed to work on yours.
```
Minor language fix + remove use of gratuitous semicolon.

**Review by gabrielskoglund — COMMENTED** [review]
Well written report! I added a few minor language nitpicks but otherwise I think it's good to merge.

**Comment by algomaster99** [comment]
I usually prefer to restrict line length to `80` characters because it just makes it more readable in the file view. However, I cannot format it here and you may choose not to as well because it will be deployed anyway.

Excellent report overall! :)

File: nixcon-2022.md (hunk 1 above)
Comment (monperrus) [inline nixcon-2022.md:5]: add a link to the dissertation

File: nixcon-2022.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (monperrus) [inline nixcon-2022.md:6]: would be great to have an illustrative listing of Nix code

File: nixcon-2022.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Reproducibility is not all about software security. Martin Kowalewski of Stockholm University gave [a talk](https://talks.nixcon.org/nixcon-2022/talk/MYHSKT/) about how he uses Nix as a part of his research in computational chemistry. The high performance computing software that underpins his research is often made in house at the academic institutions that uses them. The programs tend to grow organically over many years, as demands change. Often, however, maintainability is not the primary concern of the programmers. They are chemists that need to run an experiment, not software engineers. Since the code tends to be quite low level, reproducibility quickly becomes a concern. The output may depend subtly on the exact versions of libraries for scientific computations, and sometimes even on the hardware on which it is executed. Prof. Kowalewski explained that Nix allowed him to eliminate variable factors, which increased the reproducibility and maintainability of his software. In other words, Nix helped scientific experiments to be reproducible.
```
Comment (monperrus) [inline nixcon-2022.md:15]: do you mean "not only" instead of "not al all"?

File: nixcon-2022.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Not all talks were given by men named Martin. These were just a few highlights. A complete schedule with all talks and speakers can be found [here](https://talks.nixcon.org/nixcon-2022/schedule/).
+
```
Comment (monperrus) [inline nixcon-2022.md:18]: Could you finish with the two/three main take aways (bullet points)

And maybe a short speculative paragraph on what the CHAINS ideas could bring to NIX?

**Review by monperrus — COMMENTED** [review]
Thanks a lot for the first version @siberov see my comments

**Pull request merged**